### ISSUE TITLE
Configuration from dotenv not used

### ISSUE THREAD

**Issue by Learloj** [prestix-studio/cloudns_api#9]
I ran into an issue where I wanted to use dotenv as suggested in the README.md to set configuration settings, but this didn't seem to work.

In the end I figured out this is due to the order in which the configuration is loaded and set.

My PoC code:
```
import os
import cloudns_api
from dotenv import load_dotenv

load_dotenv()

print('Environment setting: %s' % os.environ.get('CLOUDNS_API_DEBUG'))
print('ClouDNS API setting: %s' % cloudns_api.config.CLOUDNS_API_DEBUG)
```

Because cloudns_api is loaded at the top, which loads api.py which loads config.py, all configuration variables are set from the environment settings present at that time. As they are not set, they get their default values (ie. CLOUDNS_API_DEBUG is False). Only after this has happened do we run load_dotenv() to load the .env settings into environment variables, but as those are not re-read by cloudns_api, the cloudns_api config values remain default and not the settings from my .env file.

A workaround for now could be that we disregard PEP-8 and import the clouddns_api module after load_dotenv(), which is not really desired.

Am I missing another option to re-read the configuration settings, or to postpone loading them?

If not, would it be an idea to rewrite the configuration code of this module a bit, so the configuration is read at the moment a function requiring configuration is called, instead of upon import?

If desired, I'd be open to creating a PR to make a suggested implementation of this.

**Comment by hbradleyiii** [comment]
Good catch. I guess I haven't actually used dotenv with this API yet. I just assumed it would work. 

Yeah, the workaround isn't ideal. 

If you have the time, go ahead and submit a PR.

**Comment by hbradleyiii** [comment]
Fixed with #10
